samples/dnn/ssd_mobilenet_object_detection.cpp

```diff
@@ -55,19 +58,40 @@ int main(int argc, char** argv)
     dnn::Net net = readNetFromCaffe(modelConfiguration, modelBinary);
     //! [Initialize network]
 
-    VideoCapture cap(parser.get<String>("video"));
-    if(!cap.isOpened()) // check if we succeeded
+    if (net.empty())
     {
-        cap = VideoCapture(0);
-        if(!cap.isOpened())
+        cerr << "Can't load network by using the following files: " << endl;
+        cerr << "prototxt:   " << modelConfiguration << endl;
+        cerr << "caffemodel: " << modelBinary << endl;
+        cerr << "Models can be downloaded here:" << endl;
+        cerr << "https://github.com/chuanqi305/MobileNet-SSD" << endl;
+        exit(-1);
+    }
+
+    Mat img;
+    VideoCapture cap;
+    if (parser.get<String>("image").empty()) {
```

Comment:
`parser.get<String>("image")` used here 5 times in this example. Probably we could save this value into some variable:
```
String imagePath = parser.get<String>("image");
if (imagePath.empty())
    ...
```

Reply:
We also can  pass image path to VideoCapture. Extra waitKey might be after the last frame processed.